articles/communication-services/concepts/voice-video-calling/calling-sdk-features.md

```diff
@@ -51,20 +51,45 @@ The following list presents the set of features which are currently available in
-| General           | Test your mic, speaker, and camera with an audio testing service (available by calling 8:echo123)                   |  ✔️  | ✔️            | ✔️   
+| General           | Test your mic, speaker, and camera with an audio testing service (available by calling 8:echo123)                   | ✔️   | ✔️            | ✔️ 
+| Device Management | Ask for permission to use  audio and/or video                                                                       | ✔️   | ✔️            | ✔️
+|                   | Get camera list                                                                                                     | ✔️   | ✔️            | ✔️ 
+|                   | Set camera                                                                                                          | ✔️   | ✔️            | ✔️
+|                   | Get selected camera                                                                                                 | ✔️   | ✔️            | ✔️
+|                   | Get microphone list                                                                                                 | ✔️   | ✔️            | ✔️
+|                   | Set microphone                                                                                                      | ✔️   | ✔️            | ✔️
+|                   | Get selected microphone                                                                                             | ✔️   | ✔️            | ✔️
+|                   | Get speakers list                                                                                                   | ✔️   | ✔️            | ✔️
+|                   | Set speaker                                                                                                         | ✔️   | ✔️            | ✔️
+|                   | Get selected speaker                                                                                                | ✔️   | ✔️            | ✔️
+| Video Rendering   | Render single video in many places (local camera or remote stream)                                                  | ✔️   | ✔️            | ✔️
+|                   | Set / update scaling mode                                                                                           | ✔️   | ✔️            | ✔️ 
+|                   | Render remote video stream                                                                                          | ✔️   | ✔️            | ✔️
+
+
 
 ## JavaScript calling client library support by OS and browser
 
-The following table represents the set of supported browsers and versions which are currently available.
+The following table represents the set of supported browsers and versions which are currently available. We support the most recent three versions unless otherwise indicated.
+
+|                                  | Chrome | Safari*  | Edge(Chromium) | 
```

Comment:
We have verion of the Safari supported, what about Chrome and Edge (Chromium)?

Reply:
@nmurav The latest three revisions are supported for Chrome and Edge(Chromium), the exception here is Safari where we we are not supporting Safari 12. Moving forward as new versions of the browsers evolve, our commitment will be to support the latest three versions unless otherwise stated.  
Chrome (86,87,88), 
Safari (14,13,12)
Edge(Chromium) 86,87,88